src/geo/points.ts

```diff
@@ -72,7 +79,34 @@ const distance = (pointFrom: Point, pointTo: Point): number | null => {
   return earthRadius * c
 }
 
+const equals = (point1: Point, point2: Point): boolean => {
+  if (point1 === null && point2 === null) return true
+  if (point1 === null || point2 === null) return false
```

Comment:
can this happen ?

Reply:
actually it cannot happen, point1 and point2 are mandatory... I'll remove that check